ding2.install

```diff
@@ -306,3 +306,10 @@ function ding2_update_7012() {
   $eu_cookie_compliance_da['popup_disagree_button_message'] = 'Læs mere';
   variable_set('eu_cookie_compliance_da', $eu_cookie_compliance_da);
 }
+
+/**
+ * Enable manualcrop module.
+ */
+function ding2_update_7013() {
+  module_enable(array('manualcrop'));
+}
```

Comment:
If this module is required by `ding_fronten` then the install hook for enabling it should also be placed there.